### PR TITLE
Extract content presenter out of routes.js

### PR DESCRIPTION
**Pull request by mobaig**
Place logic handling presentation of a content page into its own module.
Makes routes.js simpler and the resulting module easier to refactor.

This PR also adds a macro to handle breadcrumb rendering.

https://trello.com/c/CcSGe6qa/241-rework-content-route-moving-some-of-its-logic-out-into-one-or-more-modules

### CODE REVIEW

File: app/models/content_presenter.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      }).
+      then(function (fileInfo) {
+        const content = fileInfo.fileData.toString();
+        const filePath = fileInfo.filePath;
+
+        var isHtmlPublication = content.match(/html-publications-show/);
+        var isWhitehall = filePath.match(/whitehall/);
+        var isHtmlManual = filePath.match(/manual/);
+
+        if (!isHtmlPublication) {
```
Comment (carvil) [inline app/models/content_presenter.js]: I don't think we will have HTML publications for education. Should we delete the logic?

File: app/models/content_presenter.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+              isWhitehall: isWhitehall,
+              isHtmlManual: isHtmlManual,
+              isHtmlPublication: isHtmlPublication,
+            }
+          })
+
+          return presented;
+        }
+
+        // Skip breadcrumbs and taxons for HTML publications since they have a
```
Comment (carvil) [inline app/models/content_presenter.js]: Same goes here, I think we can delete this comment.

File: app/models/content_presenter.js (hunk 1 above)
Comment (mobaig) [inline app/models/content_presenter.js]: [Baleeted.](https://www.youtube.com/watch?v=07h0ksKx5sM)

**Comment by carvil** [comment]
👍

**Pull request merged**